### DISCUSSION TITLE
[General] What does "standard policies are too restrictive" in case of the SnapCraft Store mean?

### DISCUSSION THREAD

**Discussion by AndiMb** [glpi-project/glpi-agent#445] [General]
### Your question

Hi,

I'm having a question about a hint in the documentation (https://glpi-agent.readthedocs.io/en/latest/installation/index.html#snap). 

Why are the Snapcraft store standard policies too restrictive for the GLPI Agents requirements and features to provide a package of the GLPI Agent there?

Regards,
Andreas

**Comment by g-bougard** [comment]
Hi @AndiMb 

the fact is the inventory case was not included in the Snapcraft store standard policies.

You can check the full history of my exchanges 2 years ago with the snapcraft team where you should fine your answer:
https://forum.snapcraft.io/t/request-classic-confinement-for-glpi-agent/22304
